ziro-cdr-template.csv

```diff
@@ -0,0 +1,4 @@
+Source Phone Number,Source user,Destination Phone Number,Destination user,Call Duration (in seconds),Start time (in ISO8601)
+5001,bdesnoyers,5002,ckarawani,525,2024-10-29T05:50:12Z
+5001,bdesnoyers,,+15149408284,384,2024-10-29T05:50:12Z
+5002,ckarawani,,+15149408000,1084,2024-10-29T05:50:12Z
```

Comment:
> Upload a copy of the CSV file template to our pbx-exporter repo under ziro-cdr-template.csv.

Did benoit want example values in the template or an empty template? I'm not sure from the success criteria